src/components/MobileNav.tsx

```diff
@@ -0,0 +1,110 @@
+import { FC } from 'react'
+import useSupabaseAuth from '../hooks/useSupabaseAuth'
+import { version } from "../../package.json";
+import { capturePostHogAnayltics } from "../lib/analytics";
+import { Menu } from '@headlessui/react'
+import { GiHamburgerMenu } from 'react-icons/gi';
+
+
+
+const navs = [
+  {
+    placeholder: 'Docs',
+    links: "https://docs.opensauced.pizza/"
+  },
+  {
+    placeholder: 'Blog',
+    links: "https://dev.to/opensauced"
+  },
+  {
+    placeholder: 'Github',
+    links: "https://github.com/open-sauced"
+  },
+  {
+    placeholder: 'YouTube',
+    links: "https://www.youtube.com/opensauced"
+  }
+]
+
+const MobileNav:FC = () => {
+  const { signIn, signOut, user } = useSupabaseAuth();
+
+  return (
+    <div className=' tablet:hidden'>
+      <Menu as="div" className="relative">
+        <Menu.Button>
+          <GiHamburgerMenu className='w-[35px] h-[35px] text-gray-600 ' />
+        </Menu.Button>
+
+        <Menu.Items className="absolute z-50 right-0 w-56 origin-top-right rounded-md shadow-sm border-gray-100 border-[1px] focus:outline-none px-1 py-1 bg-white text-sm font-semibold">
+          {
+            navs.map( nav => (
+              <Menu.Item key={nav.placeholder}>
+                <div className='bg-w p-[5px] text-[14px] text-gray150 rounded-md hover:bg-gray100 '>
```

Comment:
Would not be p-2 ? not p-5px?

Reply:
@NsdHSO, I think p-[5px] is more specific.

Reply:
indeed p-[5px] is more specific, would not have to write here your custom class ?

Reply:
@NsdHSO tailwind now automatically consider these arbitrary values as a valid CSS already.